flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-doris/src/main/java/com/ververica/cdc/connectors/doris/sink/DorisDataSinkOptions.java

```diff
@@ -0,0 +1,167 @@
+            ConfigOptions.key("password")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription("the doris password.");
+    public static final ConfigOption<String> JDBC_URL =
+            ConfigOptions.key("jdbc-url")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription("doris jdbc url address.");
+    public static final ConfigOption<Boolean> AUTO_REDIRECT =
+            ConfigOptions.key("auto-redirect")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription(
+                            "Use automatic redirection of fe without explicitly obtaining the be list");
+
+    // Streaming Sink options
+    public static final ConfigOption<Boolean> SINK_ENABLE_2PC =
+            ConfigOptions.key("sink.enable-2pc")
+                    .booleanType()
+                    .defaultValue(true)
+                    .withDescription("enable 2PC while loading");
+
+    public static final ConfigOption<Integer> SINK_CHECK_INTERVAL =
+            ConfigOptions.key("sink.check-interval")
+                    .intType()
+                    .defaultValue(10000)
+                    .withDescription("check exception with the interval while loading");
+    public static final ConfigOption<Integer> SINK_MAX_RETRIES =
+            ConfigOptions.key("sink.max-retries")
+                    .intType()
+                    .defaultValue(3)
+                    .withDescription("the max retry times if writing records to database failed.");
+    public static final ConfigOption<Integer> SINK_BUFFER_SIZE =
+            ConfigOptions.key("sink.buffer-size")
+                    .intType()
+                    .defaultValue(1024 * 1024)
+                    .withDescription("the buffer size to cache data for stream load.");
+    public static final ConfigOption<Integer> SINK_BUFFER_COUNT =
+            ConfigOptions.key("sink.buffer-count")
+                    .intType()
+                    .defaultValue(3)
+                    .withDescription("the buffer count to cache data for stream load.");
+    public static final ConfigOption<String> SINK_LABEL_PREFIX =
+            ConfigOptions.key("sink.label-prefix")
+                    .stringType()
+                    .defaultValue("")
+                    .withDescription("the unique label prefix.");
+    public static final ConfigOption<Boolean> SINK_ENABLE_DELETE =
+            ConfigOptions.key("sink.enable-delete")
+                    .booleanType()
+                    .defaultValue(true)
+                    .withDescription("whether to enable the delete function");
+
+    // batch sink options
+    public static final ConfigOption<Boolean> SINK_ENABLE_BATCH_MODE =
+            ConfigOptions.key("sink.enable.batch-mode")
+                    .booleanType()
+                    .defaultValue(false)
```

Comment:
As we don't support two phase commit current in FlinkCDC framework, can we set the default value to `true`? It will cause inconsistence if we don't commit transaction after SinkWriter flush.

Reply:
Thanks for the reminder, modified

Reply:
already modified